pysat/utils/_core.py

```diff
@@ -0,0 +1,410 @@
+from __future__ import print_function
+from __future__ import absolute_import
+
+import numpy as np
+
+
+def set_data_dir(path=None, store=None):
+    """
+    Set the top level directory pysat uses to look for data and reload.
+
+    Parameters
+    ----------
+    path : string
+        valid path to directory pysat uses to look for data
+    store : bool
+        if True, store data directory for future runs
+    """
```

Comment:
Need a blank line before the end of the docstring.  Should also have Return (even if it says Void)

Reply:
fixed throughout.